pkg/model/api/auth.go

```diff
@@ -43,12 +43,12 @@ func (u *Auth) UnmarshalJSON(data []byte) error {
 }
 
 type AuthResponse struct {
-	AccessToken  string `json:"accessToken"`
```

Comment:
Could you add a note saying the json field name format is respecting the Oauth/OIDC Spec which is different from the convention we have for json field in Perses ?

Reply:
Commented the struct